python/GafferUI/FileMenu.py

```diff
@@ -51,6 +51,7 @@ def appendDefinitions( menuDefinition, prefix="" ) :
 	menuDefinition.append( prefix + "/New", { "command" : new, "shortCut" : "Ctrl+N" } )
 	menuDefinition.append( prefix + "/Open...", { "command" : open, "shortCut" : "Ctrl+O" } )
 	menuDefinition.append( prefix + "/Open Recent", { "subMenu" : openRecent } )
+	menuDefinition.append( prefix + "/Open Backup", { "command" : openBackup, "active" : __backupAvailable } )
```

Comment:
I think this menu item probably makes more sense below "Revert to Saved", since it is a similar operation, and acts relative to _this_ script (whereas all of the New/Open items don't care about the current script).

Reply:
I also wonder, would it be useful to have access to all the backups, in a submenu, sorted and labelled by date?

Reply:
I too was thinking that, but I don't want to complicate this PR any more and that could be done in a later date. I can add a TODO somewhere in the code to report this.